app/views/organisations/_org_contacts.html.erb

```diff
@@ -8,6 +8,6 @@
       lang: t_fallback("organisations.contact.contact_organisation", { organisation: @show.acronym })
   } %>
 
-  <%= render partial: 'contacts', locals: { contacts: contacts.contacts, border: true } %>
+  <%= render partial: 'contacts', locals: { contacts: contacts.contacts, border: true, ga4_section: t('organisations.contact.contact_organisation', organisation: @show.acronym, lang: :en) } %>
```

Comment:
I'm not familiar with this code so just wanted to check - does this value for section need to be passed here, or could it just be included directly in `_contacts.html.erb`?

Reply:
@andysellick Good that you mentioned this - it will need to be structured like this as there's two places this partial is rendered, and they both need to pass in a different section. Good that you mentioned this as I had missed out adding the `section` to the other partial render. I've added that in the latest commit.

Reply:
Excellent glad to know my ignorance is useful for something 😁 good spot